corehq/apps/products/migrations/0001_initial.py

```diff
@@ -25,7 +25,7 @@ class Migration(migrations.Migration):
                 ('program_id', models.CharField(default=b'', max_length=100, null=True)),
                 ('cost', models.DecimalField(null=True, max_digits=20, decimal_places=5)),
                 ('units', models.CharField(default=b'', max_length=100, null=True)),
-                ('product_data', json_field.fields.JSONField(default={}, help_text='Enter a valid JSON object')),
+                ('product_data', jsonfield.fields.JSONField(default={})),
```

Comment:
http://stackoverflow.com/questions/26503826/why-does-django-generate-migrations-for-help-text-and-verbose-name-changes?answertab=votes#tab-top